ui/tests/acceptance/secrets/backend/kv/kv-v2-workflow-navigation-test.js

```diff
@@ -164,7 +168,9 @@ module('Acceptance | kv-v2 workflow | navigation', function (hooks) {
       );
     });
     test('can access nested secret (a)', async function (assert) {
-      assert.expect(46);
+      // enterprise has "Patch latest version" in the toolbar which adds an assertion
```

Comment:
I don't love doing this, but working with how this test was written. I'd like to refactor the `assertDetailsToolbar` function to be more helpful and iterate over all of the existing items in the toolbar rather than a static list of `DETAIL_TOOLBARS` strings. 

This way we'll catch any additional actions that are otherwise missed.